pysilcam/process.py

```diff
@@ -385,26 +391,27 @@ def extract_particles(imc, timestamp, settings, nnmodel, class_labels, region_pr
         data[i, :] = [getattr(el, p) for p in propnames]
         bboxes[i, :] = el.bbox
 
-        # if operating in realtime mode, assume we only care about oil and gas and skip export of overly-derformed particles
-        if settings.Process.real_time_stats & (((data[i, 1]/data[i, 0])<0.3) | (data[i, 3]<0.95)):
+        # if operating in realtime mode, assume we only care about oil and gas and skip export of overly-derformed
+        # particles
+        if settings.Process.real_time_stats & (((data[i, 1] / data[i, 0]) < 0.3) | (data[i, 3] < 0.95)):
             continue
         # Find particles that match export criteria
-        if ((data[i, 0] > settings.ExportParticles.min_length) & #major_axis_length in pixels
-            (data[i, 1] > 2)): # minor length in pixels
+        if ((data[i, 0] > settings.ExportParticles.min_length) &  # major_axis_length in pixels
+                (data[i, 1] > 2)):  # minor length in pixels
 
             nb_extractable_part += 1
             # extract the region of interest from the corrected colour image
-            roi = extract_roi(imc,bboxes[i, :].astype(int))
+            roi = extract_roi(imc, bboxes[i, :].astype(int))
 
             # add the roi to the HDF5 file
             filenames[int(i)] = filename + '-PN' + str(i)
             if settings.ExportParticles.export_images:
-                dset = HDF5File.create_dataset('PN' + str(i), data = roi)
-                #@todo also include particle stats here too.
+                dset = HDF5File.create_dataset('PN' + str(i), data=roi)
```

Comment:
F841: local variable 'dset' is assigned to but never used  <!-- Automatically posted by Lintly -->